uni/lib/view/restaurant/cantine_page_view.dart

```diff
@@ -0,0 +1,176 @@
+import 'package:flutter_redux/flutter_redux.dart';
+import 'package:tuple/tuple.dart';
+import 'package:uni/model/app_state.dart';
+import 'package:uni/model/entities/meal.dart';
+import 'package:flutter/material.dart';
+import 'package:uni/view/common_widgets/page_title.dart';
+import 'package:uni/view/common_widgets/pages_layouts/general/general.dart';
+import 'package:uni/model/utils/day_of_week.dart';
+
+import 'package:uni/model/entities/restaurant.dart';
+import 'package:uni/view/common_widgets/request_dependent_widget_builder.dart';
+import 'package:uni/view/restaurant/widgets/cantine_slot.dart';
+import 'package:uni/view/theme.dart';
+
+class CantinePageView extends StatefulWidget {
+  const CantinePageView({Key? key}) : super(key: key);
+
+  @override
+  State<StatefulWidget> createState() => _CantinePageState();
+}
+
+class _CantinePageState extends GeneralPageViewState<CantinePageView>
+    with SingleTickerProviderStateMixin {
+
+  final List<DayOfWeek> daysOfTheWeek = [
+    DayOfWeek.monday,
+    DayOfWeek.tuesday,
+    DayOfWeek.wednesday,
+    DayOfWeek.thursday,
+    DayOfWeek.friday,
+    DayOfWeek.saturday,
+    DayOfWeek.sunday
+  ];
+
+  late List<Restaurant> aggRestaurant;
+  late TabController tabController;
+  late ScrollController scrollViewController;
+
+  @override
+  void initState() {
+    super.initState();
+    final int weekDay = DateTime.now().weekday;
+    super.initState();
+    tabController = TabController(vsync: this, length: daysOfTheWeek.length);
+    final offset = (weekDay > 5) ? 0 : (weekDay - 1) % daysOfTheWeek.length;
+    tabController.animateTo((tabController.index + offset));
+    scrollViewController = ScrollController();
+  }
+
+  @override
+  Widget getBody(BuildContext context) {
+    return StoreConnector<AppState, Tuple2<List<Restaurant>, RequestStatus?>>(
+        converter: (store) {
+          return Tuple2(store.state.content['restaurants'],
+              store.state.content['restaurantsStatus']);
+        },
+        builder: (context, restaurantsInfo) =>
+            _getPageView(restaurantsInfo.item1, restaurantsInfo.item2));
+
+  }
+
+  Widget _getPageView(List<Restaurant> restaurants, RequestStatus? status) {
+    if (status == null) {
```

Comment:
Do we need this check? All request related logic should be handled by `RequestDependantWidgetBuilder`